_schemaV2.graphql

```diff
@@ -4665,7 +4665,7 @@ type CreditCard {
   # A globally unique ID.
   id: ID!
 
-  # A type-specific ID likely used as a database ID.
+  # A type-specific ID.
   internalID: ID!
```

Comment:
Other than comments I’m not seeing any actual changes in the schema, so this doesn’t appear to actually do anything. Perhaps you were initially working on the codebase that was still using transformations for the ID fields (which identified the fields based on their description)?

Reply:
The schemes is unchanged, it’s the resolver that’s different.